server/chat.ts

```diff
@@ -1293,17 +1293,18 @@ export class CommandContext extends MessageContext {
 					if ((!this.room || this.room.settings.isPersonal || this.room.settings.isPrivate === true) && !this.user.can('lock')) {
 						const buttonName = / name ?= ?"([^"]*)"/i.exec(tagContent)?.[1];
 						const buttonValue = / value ?= ?"([^"]*)"/i.exec(tagContent)?.[1];
-						if (buttonName === 'send' && buttonValue?.startsWith('/msg ')) {
-							const [pmTarget] = buttonValue.slice(5).split(',');
+						const msgCommandRegex = /^\/(?:msg|pm|w|whisper) /i;
+						if (buttonName === 'send' && msgCommandRegex.test(buttonValue || '')) {
+							const [pmTarget] = buttonValue.replace(msgCommandRegex, '').split(',');
```

Comment:
```suggestion
						const msgCommandRegex = /^\/(?:msg|pm|w|whisper) /i;
						if (buttonName === 'send' && buttonValue && msgCommandRegex.test(buttonValue)) {
							const [pmTarget] = buttonValue.replace(msgCommandRegex, '').split(',');
```

Not sure why it's so low, but your check failure is coming from here. RegExp.exec can return null, so you need to assert that it exists - the ``|| ''`` isn't what you want to do there.

Reply:
Yeah, this was my mistake thinking through the logic wrong.